watershed-android/watershed/app/src/main/java/com/blueprint/watershed/MiniSites/MiniSiteFragment.java

```diff
@@ -99,7 +99,7 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
     public void onResume() {
         super.onResume();
         mParentActivity.setMenuAction(false);
-        getMiniSiteRequest(mMiniSite);
+//        getMiniSiteRequest(mMiniSite);
```

Comment:
Are we no longer updating when the app re-opens? That would make sense, and if it's the case, then we can just get rid of this line entirely.